uint32 trigger_count
baslerStatus[] status
